lib/vyasa_web/live/source_live/player/player_live.ex

```diff
@@ -604,18 +621,18 @@ defmodule Vyasa.Player.PlayerLive do
   #   # end
   # end
 
-  defp push_play(socket, # %Song{} =
-    _song, elapsed) do
-    song = @stub_song
+  defp push_play(socket,  %Song{} =
+    song, elapsed) do
     token = Phoenix.Token.encrypt(socket.endpoint, "file", %{
```

Comment:
this is the problem child, 

the way i added in a stub song was to use a locally runnign instance of live_beats, then uploading there 

adn then using that reference to create the song stub. 

the problem with doing so is that this encrypt fn won't create the correct token.